tools/verify-commit/verify-commit.go

```diff
@@ -34,13 +34,21 @@ type GitCommit struct {
 	DCOCoauthorTag   string `json:"dcoCoauthorTag"`
 }
 
-const dependabot = "dependabot[bot]"
+const (
+	dependabot = "dependabot[bot]"
+	konflux    = "red-hat-konflux"
```

Comment:
will this work for "red-hat-konflux[bot]" author as well ?

Reply:
yes because this is not looking to find exact author name but it is enough if it contains it:
https://github.com/openshift-kni/numaresources-operator/pull/1298/files#diff-f0ad47dc1690100402944f42e006fc4f7fdd449f6a4658415553d6781325c942R48